### ISSUE TITLE
Sync First Name, Last Name, Org, and Interests with MailChimp

### ISSUE THREAD

**Issue by hrod** [badcamp/badcamp-2017-Sprint-Planning#88]

**Comment by hrod** [comment]
- Interests need a little bit more investigation.

**Comment by shirazd** [comment]
I added first and last name sync to their existing fields on the Mailchimp side. 

For Organization, I had to add the field on the mailchimp side, then I added sync. 

I have been granted full perms on the mailchimp side, and I confirmed that the sync was working a https://us2.admin.mailchimp.com/lists/members/?id=776509#p:1-s:10-sa:last_update_time-so:false. 

Interests, I am not doing anything for. I think it was my suggestion to potentially add this in the first place, but I was mixing up the List Groups with the Interests. List Groups, best I can tell, is the ability to define sublists of the main list. So we would do that on Mailchimp side and then on the drupal side peeps can choose their sublists that they want to be subscribed to, but that's overkill. By contrast, the Interests field we are using on Drupal is a free form tax field, which I don't figure is worth syncing on the Mailchimp side, but if you think otherwise, send this back to me. I'm not sure how simple this would be though because on Mailchimp's field config UI (https://us2.admin.mailchimp.com/lists/designer/?id=776509) I don't see how to add multi val fields, plus I'd probably have to hack into the mailchimp module. Probably not worth it. Again, I *think* I was the one that even made this sync suggestion in the first place, based on a misunderstanding.
